tools/java/org/unicode/cldr/api/CldrValue.java

```diff
@@ -58,6 +59,41 @@ public static CldrValue parseValue(String fullPath, String value) {
         return new CldrValue(value, valueAttributes, path);
     }
 
+    /**
+     * Returns a value whose path has been replaced with the specified distinguished path. Note
+     * that the specified path must have the same element names as the path of the original value,
+     * though it can differ with regard to distinguishing attributes.
+     *
+     * <p>In general it is not safe or meaningful to think of replacing arbitrary paths for a value,
+     * but when the path are "in the same location" in the hierarchy then it is safe because a
+     * path's namespace is what identifies the semantic meaning of a value.
+     *
```

Comment:
To be safer, it should only handle paths that differ by attributes on the _final_ element. (differing attributes in the higher elements can indicate very different functions)

Reply:
Hmm. Perhaps "the final element which can have attributes" since many paths have an "empty" leaf element (since they are part of a list). However this gets complex since all elements can theoretically have "draft" (and possibly others?) so I'd need a bit of careful checking.

Since this is mostly needed to avoid syntactic issues (moving value attributes into a path which doesn't contain the associated element) and it currently works for that case, I'm happy to submit now with an issue raises/TODO added.

I'll dig a bit more as to where the "leaf elements" rule might be an issue (this could even be limited to only affecting "alt" attributes, but I think that's a bit limiting in the general API.

Reply:
I think the note: 
     * In general it is not safe or meaningful to think of replacing arbitrary paths for a value,
     * but when the path are "in the same location" in the hierarchy then it is possible. Care must
     * still be taken to ensure that the source and target paths are semantically interchangeable.
is still misleading. It makes it sound like the proviso in the first paragraph ("must have the same element names as the path of the original value") is a guarantee of they are "in the same location", which is wrong.

Reply:
Please reword it to your satisfaction via the web UI when you get a moment (you should be able to edit it directly).

Reply:
Don't see how to edit your PR, but here is suggested wording. See what you think.

/**
     * Returns a value whose path has been replaced with the specified distinguished path. 
     * <p>In general, it is not safe to change paths arbitrarily. Care must
     * be taken to ensure that the source and target paths are semantically interchangeable.
     * <p>A very basic test is in place to prevent the most egregious errors, by ensuring that the
     * replacement path has the same <i>elements</i> as the original, but allowing attributes and their
     * values to be different. Do not, however, depend upon that test catching all problems.